.github/workflows/add_contributor.yml

```diff
@@ -0,0 +1,51 @@
+name: Add Contributor to AUTHORS.md
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  add_contributor:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Check if contributor already exists
+        id: check_contributor
+        run: |
+          exists=$(grep -c -F "$GITHUB_ACTOR" AUTHORS.md)
+          if [[ $exists -gt 0 ]]; then
+            echo "::set-output name=exists::true"
+          else
+            echo "::set-output name=exists::false"
+          fi
+
+      - name: Get contributor's full name
+        id: get_full_name
+        if: steps.check_contributor.outputs.exists == 'false'
+        run: |
+          username=$GITHUB_ACTOR
+          access_token=$GITHUB_TOKEN
+          full_name=$(curl -s -H "Authorization: token $access_token" "https://api.github.com/users/$username" | jq -r '.name')
+          if [[ -z "$full_name" ]]; then
+            full_name=$username
+          fi
+          echo "::set-output name=full_name::$full_name"
+
+      - name: Append contributor to AUTHORS.md
+        id: append_contributor
+        if: steps.check_contributor.outputs.exists == 'false'
+        run: |
+          echo "$GITHUB_ACTOR - ${{ steps.get_full_name.outputs.full_name }}" >> AUTHORS.md
```

Comment:
our [AUTHORS.md](https://github.com/learningequality/kolibri/blob/develop/AUTHORS.md) file is in a specific format we'd like to maintain: a new table row for each entry with a first column of `FULL_NAME` & a second column of `USERNAME`, in that specific order. could you update the output here to reflect that order & include the `|`s necessary for proper formatting? 

for instance, [the code that produces the table row for me](https://github.com/learningequality/kolibri/blob/a196c1ee581481670ca1400df1ccaf6bf8ba8053/AUTHORS.md?plain=1#L98) is `| Amelia Breault | thanksameeelian |`